src/gluonts/model/predictor.py

```diff
@@ -501,10 +477,9 @@ def as_symbol_block_predictor(
             freq=self.freq,
             ctx=self.ctx,
             input_transform=self.input_transform,
+            forecast_wrapper=self.forecast_wrapper,
             output_transform=self.output_transform,
             float_type=self.float_type,
-            forecast_cls_name=self.forecast_cls_name,
-            forecast_kwargs=self.forecast_kwargs,
```

Comment:
The `_kwargs` also need to be forwarded

Reply:
Thanks, good point. Renamed it to `ForecastGenerator`.

Also I just realized that the `forecast_kwargs` is not needed anymore, since such fields can be passed to the constructor of the specific `ForecastGenerator` as is done e.g. for the `QuantileForecast`.